tsconfig.spec.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "./tsconfig",
   "compilerOptions": {
-    "allowJs": false,
-    "checkJs": false
+    "allowJs": true,
+    "checkJs": true,
+    "types": ["node", "jest", "jest-extended", "expect-more-jest"]
```

Comment:
revert

Reply:
Yes by mistake. Will order things now

Reply:
reverted

Reply:
https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting 🤗